src/components/CardView.ts

```diff
@@ -1,24 +1,24 @@
-import { Card } from "../deck";
-import { Config } from '../config';
+import { Card } from "../deck/deck";
```

Comment:
@zveillette When that happens, if you want to avoid the double deck/deck, you can rename the deck.ts file to index.ts, and to import it, you can now do "../deck"

Reply:
Good to know, I'll leave as is for the moment. Was just regrouping them, and deck isn't really the "index" of all these modules.  I'd imagine it would be a pain to quickly open file if you have multiple index files also ahaha

Reply:
index1.ts, index2.ts, index3.ts